app.js

```diff
@@ -1,4 +1,13 @@
-import { Product, User } from './models';
+import Log from 'log';
+import { DirWatcher, Importer } from './models';
 
-new Product();
-new User();
+const dirWatcher = new DirWatcher();
+const log = new Log();
```

Comment:
Maybe it makes sense to create one log instance for all modules?

Reply:
Right. I've created log.js for that